br/pkg/lightning/restore/restore.go

```diff
@@ -263,6 +264,8 @@ type ControllerParam struct {
 	CheckpointStorage storage.ExternalStorage
 	// when CheckpointStorage is not nil, save file checkpoint to it with this name
 	CheckpointName string
+	// DupIndicator can expose the duplicate detection result to the caller
```

Comment:
```suggestion
	// DupIndicator can expose the duplicate detection result to the caller (dm)
```
If this feature is only used by DM, I think this can make it clearer.

Reply:
Prefer not. Lightning can also be used by other projects. Any API changes are available to all users.